zookeeper/core/cli_test.py

```diff
@@ -4,9 +4,11 @@
 from zookeeper.core.cli import cli
 from zookeeper.core.component import component
 from zookeeper.core.factory import factory
-from zookeeper.core.field import ComponentField, Field
+from zookeeper.core.field import ComponentField, Field  # type: ignore
```

Comment:
This is because `pytype` has gotten smarter and now realises that `a: int = Field(5)` doesn't make sense out of context. Not sure if there's a better solution than this, though thankfully this only matters within this repo because clients importing zookeeper don't run into this problem (for now at least).